packages/backend/src/clients/EmailClient/EmailClient.ts

```diff
@@ -148,6 +148,7 @@ export default class EmailClient {
         let roleAction = 'view';
         switch (projectMember.role) {
             case ProjectMemberRole.VIEWER:
+            case ProjectMemberRole.INTERACTIVE_VIEWER:
```

Comment:
The copy on this email is actually a bit weird. I think the role actions should say:

 - view
 - explore
 - edit
 - administer

The template is "X has invited you to <action> this project"

Reply:
The template TEXT says: `X has invited you to view/edit/admin this project: ${projectUrl}`

I don't think this is related with this PR, so I'd suggest to create a separate issue if you want to change this copy.